src/rgw/rgw_rados.h

```diff
@@ -749,9 +749,9 @@ class RGWSystemMetaObj {
   int use_default(bool old_format = false);
 
 public:
-  RGWSystemMetaObj() {}
-  RGWSystemMetaObj(const string& _name): name(_name)  {}
-  RGWSystemMetaObj(const string& _id, const string& _name) : id(_id), name(_name) {}
+  RGWSystemMetaObj() : cct(NULL), store(NULL) {}
+  RGWSystemMetaObj(const string& _name): name(_name), cct(NULL), store(NULL)  {}
```

Comment:
could delegate to another constructor. see https://en.wikipedia.org/wiki/C%2B%2B11#Object_construction_improvement